packages/starksheet-webapp/src/types/provider/ChainConfig.ts

```diff
@@ -1,5 +1,5 @@
-import { ChainId } from './ChainId';
-import { ChainType } from './ChainType';
```

Comment:
I'm correcting these code styles when I see them, my IDE is configured for my personal code-style. Could be a good idea to setup a linter config with commit hooks, although it might be more complex with the monorepo structure.

I feel its quicker for me to manually correct them, but if you onboard more devs I can setup this, as well as a CI check for it